test/extensions/grpc_credentials/aws_iam/aws_iam_grpc_credentials_test.cc

```diff
@@ -0,0 +1,119 @@
+#ifdef ENVOY_GOOGLE_GRPC
```

Comment:
Is it possible to do this at the Bazel level vs. as an ifdef in the code?

Reply:
Do you really need this `#ifdef`? Isn't `GRPC_CLIENT_INTEGRATION_PARAMS` is already taking care of that?

Reply:
Yeah this shouldn't need the `#ifdef` the other grpc credentials plugin integ test also had the `#ifdef`, so this was just copied from it.